### PR TITLE
Implement overlay shape attribute

### PR DESCRIPTION
**Pull request by nesterov-n**
It's for #4 
added attribute `krop_shape` with two predefined value `rect` and `oval`
Default is oval

### CODE REVIEW

File: krop/src/main/java/com/avito/android/krop/OverlayView.kt [hunk 1]
```diff
@@ -41,7 +42,17 @@ class OverlayView(context: Context) : View(context) {
...
-}
\ No newline at end of file
+}
+
+@IntDef(SHAPE_OVAL.toLong(), SHAPE_RECT.toLong())
+@Retention(AnnotationRetention.SOURCE)
+annotation class OverlayShape
+
+const val SHAPE_OVAL = 0
+const val SHAPE_RECT = 0
```
Comment (dimorinny) [inline krop/src/main/java/com/avito/android/krop/OverlayView.kt]: 0?

Comment (nesterov-n) [reply krop/src/main/java/com/avito/android/krop/OverlayView.kt]: facepalm :)
Fixed

File: krop/src/main/java/com/avito/android/krop/KropView.kt [hunk 2]
```diff
@@ -39,6 +40,7 @@ class KropView(context: Context, attrs: AttributeSet) : FrameLayout(context, att
                 offset = getDimensionPixelOffset(R.styleable.KropView_krop_offset, offset)
                 aspectX = getInteger(R.styleable.KropView_krop_aspectX, aspectX)
                 aspectY = getInteger(R.styleable.KropView_krop_aspectY, aspectY)
+                overlayShape = getInteger(R.styleable.KropView_krop_shape, SHAPE_OVAL)
```
Comment (solkin) [inline krop/src/main/java/com/avito/android/krop/KropView.kt]: Already defined `overlayShape` may be used as default value here instead of `SHAPE_OVAL`.

**Review by solkin — APPROVED** [review]

**Comment by ntoskrnl** [comment]
Are you guys gonna merge this?

**Pull request merged**